templates/generic_show_form.html.twig

```diff
@@ -39,7 +39,7 @@
 {% endif %}
 
 <div class="asset {{ bg }}">
-   {{ include('components/form/header.html.twig') }}
+   {% include 'components/form/header.html.twig' with {'in_twig': true} %}
```

Comment:
Please use the `include` function, as recommanded in [Twig docs](https://twig.symfony.com/doc/3.x/tags/include.html).
```suggestion
   {{ include('components/form/header.html.twig', {'in_twig': true}) }}
```